organized/blog/2026-07-20-manage-public-talks-easily.md

```
@@ -0,0 +1,71 @@
+slug: manage-public-talks-easily
+tags: [Congregation tips, Organized app]
+hide_table_of_contents: false
+---
+
+Between local speakers, visiting speakers from neighboring congregations, outline tracking, song coordination, and the occasional Saturday-morning cancellation, the role involves more logistics than most elders realize before they're appointed to it. Let's see how this manual work can be optimized with Organized App.
+
+![Weekend meeting speakers scheduling with Organized app](./images/weekend-meeting-speakers-scheduling-organized-app.jpeg)
+
+<!-- truncate -->
+
+### What does the public talk coordinator actually manage?
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)